definitions/ext-rfscanner/dashboard.json

```diff
@@ -0,0 +1,167 @@
+{
+  "name": "Golden_Metrics_rf-device",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
+  "pages": [
+    {
+      "name": "Golden_Metrics_rf-device",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.bar"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 4
+          },
+          "title": "Total Time",
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 2811838,
```

Comment:
Since this is the first dashboard using the new one. What should we put here ? A placeholder? nothing?

Reply:
After talking with @IreneMLC we will placeholder these ones.